laurat007/assignment2/AncestorsTest.java

```diff
@@ -14,17 +14,16 @@
 import org.junit.Before;
 import org.junit.BeforeClass;
 import org.junit.Test;
-import static org.junit.Assert.*;
 
 /**
  *
  * @author Laura
  */
-public class AncestorsTest extends TestCase{
-    
+public class AncestorsTest extends TestCase {
+
     public AncestorsTest() {
```

Comment:
You don't need an empty constructor, java generates it if you haven't defined your own custom constructor.

Reply:
Removed constructor